h2/config.py

```diff
@@ -77,3 +106,61 @@ def header_encoding(self, value):
         if value is True:
             raise ValueError("header_encoding cannot be True")
         self._header_encoding = value
+
+    @property
+    def validate_sent_headers(self):
```

Comment:
Minor nit, but it would be nicer to rename this to `validate_outbound_headers` for symmetry with `validate_inbound_headers`. Alternatively, swap to `sent`/`received`.